src/Anthropic.Client/SseAggregator.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Anthropic.Client;
+
+/// <summary>
+/// Defines the base for all Aggregators using ServerSideStreaming events.
+/// </summary>
+/// <typeparam name="TMessage"></typeparam>

```

Comment:
Is there a point to defining `typeparam` in the documentation if you're not gonna document them? This seems to just be duplicative with what's already in the code